en/resources/tutorials/templating.md

```diff
@@ -16,16 +16,16 @@ redirect_from: "/resources/templating.html"
 
 Kitura is a lightweight, modular framework &mdash; this means that by default you get only basic HTTP functionality, like routing, parameter handling, parsing request body, etc. You can augment Kitura's out-of-the-box functionality by adding additional packages to your `Package.swift` file.
 
-A [template engine](https://en.wikipedia.org/wiki/Template_processor) allows rendering of documents using static templates by substituting template variables with actual values at runtime. This page explains how you can use template engines integrated with Kitura in your Kitura apps.
+Why use a template engine in your application? A [template engine](https://en.wikipedia.org/wiki/Template_processor) allows rendering of documents using static templates by substituting template variables with actual values at runtime. This means that you have clear separation between your template and data. You can also generate multiple web pages dynamically with a single template! This page explains how you can use template engines integrated with Kitura in your Kitura apps.
 
 ---
 
 ## Kitura template engines
-Kitura template engines are classes that implement the `TemplateEngine` protocol from the [Kitura-TemplateEngine package](https://github.com/IBM-Swift/Kitura-TemplateEngine/blob/master/Sources/KituraTemplateEngine/TemplateEngine.swift). Currently, three Kitura template engines are supported:
+Kitura template engines are classes that implement the `TemplateEngine` protocol from the [Kitura-TemplateEngine package](https://github.com/IBM-Swift/Kitura-TemplateEngine/blob/master/Sources/KituraTemplateEngine/TemplateEngine.swift). The most commonly used template engine is Stencil. Currently, three Kitura template engines are supported:
```

Comment:
It feels like a more natural reading would be to reverse the sentence, i.e.:
"Kitura currently supports three template engines, with Stencil being the most commonly used:"

Reply:
Reworded as it does sound a lot better that way round.